README.md

```diff
@@ -1,6 +1,60 @@
 # Ponos
-An opinonated queue based worker server for node.
+
+An opinionated queue based worker server for node.
+
+For ease of use we provide options to set the host, port, username, and password to the RabbitMQ server. If not present in options, the server will attempt to use the following environment variables and final defaults:
+
+options         | environment               | default
+----------------|---------------------------|--------------
+`opts.hostname` | `PONOS_RABBITMQ_HOSTNAME` | `'localhost'`
+`opts.port`     | `PONOS_RABBITMQ_PORT`     | `'5672'`
+`opts.username` | `PONOS_RABBITMQ_USERNAME` | `'guest'`
+`opts.password` | `PONOS_RABBITMQ_PASSWORD` | `'guest'`
+
+### Usage
+
+From a high level, Ponos is used to create a worker server that responds to jobs provided from RabbitMQ. The user defines handlers for each queue's jobs that are invoked by Ponos.
+
+Ponos has built in support for retrying and catching specific errors, which are described below.
+
+### Workers
+
+// TODO(bryan)
+
+### Example
```

Comment:
Maybe you can add a section on errors and which error should I use in each situation. Or maybe add an example of one job using all errors